src/main/java/cubicchunks/asm/mixin/fixes/common/MixinBlockFalling_HeightLimits.java

```diff
@@ -79,15 +79,15 @@ private int checkHeightYReplace3(BlockPos pos1, World worldIn, BlockPos pos) {
     
     @Redirect(method = "checkFallable", at = @At(value = "INVOKE", target = BLOCK_FALLING_CAN_FALL_THROUGH), require = 2)
     private boolean checkCanFallThrough(IBlockState blockState, World worldIn, BlockPos pos) {
-        if(((ICubicWorld)worldIn).getCubeCache().getLoadedCube(CubePos.fromBlockCoords(pos.down()))!=null) {
+        if(!((ICubicWorld)worldIn).isCubicWorld() || ((ICubicWorld)worldIn).getCubeCache().getLoadedCube(CubePos.fromBlockCoords(pos.down()))!=null) {
             return BlockFalling.canFallThrough(blockState);
         }
         return false;
     }
     
     @Redirect(method = "checkFallable", at = @At(value = "INVOKE", target = WORLD_IS_AIR_BLOCK), require = 2)
     private boolean checkIsAirBlock(World worldIn, BlockPos pos) {
-        if(((ICubicWorld)worldIn).getCubeCache().getLoadedCube(CubePos.fromBlockCoords(pos))!=null) {
+        if(!((ICubicWorld)worldIn).isCubicWorld() || ((ICubicWorld)worldIn).getCubeCache().getLoadedCube(CubePos.fromBlockCoords(pos))!=null) {
```

Comment:
Same here, you are replacing vanilla method call. For vanilla it's supposed to do exactly the same thing